### PR TITLE
Increase FirstPartyEphemeralDomainBlockStudy to 50% on nightly.

### PR DESCRIPTION
**Pull request by goodov**
https://github.com/brave/brave-browser/issues/19099

### CODE REVIEW

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers.

**Comment by iefremov** [comment]
Thanks @goodov

**Pull request merged**
